### ISSUE TITLE
failed to run the ipfs

### ISSUE THREAD

**Issue by W-B-S** [Agorise/c-ipfs#4]
lws-PC:main wsli$ ipfs daemon

No such protocol!

No such protocol!

No such protocol!

No such protocol!

I am using macOS 10.13.4 Beta (17E160e)

**Comment by marcialvieira** [comment]
There seems to be something invalid in the c-multiaddr protocols configuration, I would like to see in your config file just the multiaddr related fields: Addresses.[Swarm, API, Gateway] and Bootstrap.

**Comment by marcialvieira** [comment]
Maybe the configuration is valid, but the protocol is not yet implemented, are you using a configuration generated by the C version or the GO version?

**Comment by W-B-S** [comment]
Yeah I am using a configuration generated by GO version. should I remove the .ipfs directory and ruan ipfs init again?thanks for your reply.
